docs/pages/reference/operator-resources/resources.teleport.dev_roles.mdx

```diff
@@ -141,6 +142,12 @@ resource, which you can apply after installing the Teleport Kubernetes operator.
 |---|---|---|
 |kind|string|kind specifies the Kubernetes Resource type.|
 
+### spec.allow.request.reason
+
+|Field|Type|Description|
+|---|---|---|
+|mode|string|Mode can be either "required" or "optional". Empty-string is treated as "optional". If a role has the request reason mode set to "required", then reason is required for all Access Requests requesting roles or resources allowed by this role. It applies only to users who have this role assigned.|
```

Comment:
I don't think you need a dash for "empty string". Same for other documentation updates below.
```suggestion
|mode|string|Mode can be either "required" or "optional". Empty string is treated as "optional". If a role has the request reason mode set to "required", then reason is required for all Access Requests requesting roles or resources allowed by this role. It applies only to users who have this role assigned.|
```